### PR TITLE
Op 21619 v4.1

### PR DESCRIPTION
**Pull request by vivek-opsmx**
Added agent , argo, nameSpace field in gate service to implement Download manifest file in Cd_Integrator page
https://devopsmx.atlassian.net/browse/OP-21585

### CODE REVIEW

**Review by srikanthopsmx — APPROVED** [review]
added only new fields no functionality effect and looks fine

File: gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/OpsmxOesController.groovy [hunk 1]
```diff
@@ -510,11 +510,11 @@ class OpsmxOesController {
                                       @PathVariable("source1") String source1,
                                       @RequestParam(value = "description", required = false) String description,
                                       @RequestParam(value = "isExists", required = false) Boolean isExists,
-                                      @RequestParam(value = "namespace", required = false) String namespace,
+                                      @RequestParam(value = "nameSpace", required = false) String nameSpace,
```
Comment (jsoundar22) [inline gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/OpsmxOesController.groovy:513]: Add the PR description on why this changes required, and what is the change at high level.
and Jira details.

**Review by jsoundar22 — CHANGES_REQUESTED** [review]
